freshmaker/utils.py

```diff
@@ -217,3 +218,21 @@ def get_ocp_release_date(ocp_version):
     if not resp.ok:
         resp.raise_for_status()
     return resp.json()[0]['date_finish']
+
+
+def is_latest_eus_image(image):
+    """
+    Check if the image is EUS base image marked with 'latest' tag by checking it
+     repositories. EUS repository contains '-els' suffix.
+
+    :param ContainerImage image: image to check
+    :return: True if image is latest EUS base image, False otherwise
+    """
+    for repo in image["repositories"]:
+        if re.match(r".*-els($|\W)", repo["repository"]):
```

Comment:
By checking the repositories in Pyxis, I see there are 2 els repositories:
- rhel8-2-els/rhel
- rhel8-4-els/rhel
So you need to fix the regex pattern.

Reply:
@qixiang  Do you mean I should match only those two repos? But what if in the future we will have other repos and not only for RHEL? And if I need to match only those two repos, why can't we just have a list in that case?

Reply:
The regex pattern `r".*-els($|\W)"` doesn't match the two repositories which have `/rhel` in repository name.

Reply:
But if I execute `re.match(r".*-els($|\W)", "rhel8-2-els/rhel")` it matches, or do you mean something else?

Reply:
oops, you're right, I was checking with Pyxis's "repositories" endpoint with filter without quoting the regex pattern, sorry about that.